packages/babel-plugin-proposal-pipeline-operator/test/fixtures/fsharp/basic/output.js

```diff
@@ -0,0 +1,5 @@
+var _;
+
+var inc = x => x + 1;
+
+expect((_ = 10, inc(_))).toBe(11);
```

Comment:
If at least one of the following conditions is true, we could optimize it to `inc(10)`:
- the right expression is an identifier and it is declared
- the left expression is immutable

Reply:
Agreed. Will do.

Reply:
This also applies to minimal. Right?

Reply:
Added this in 63b7c04. I'm marking the PR as WIP, though. I'm uncomfortable with `buildOptimizedSequenceExpression` now no longer returning a sequence expression. And it also does some AST deconstructing that I would like to avoid.

Reply:
If you prefer, we can work on this optimization in a separate PR, and keep the current one as it was.